test/e2e/install/install_test.go

```diff
@@ -252,8 +254,10 @@ var _ = Describe("[Install] durability", func() {
 					klog.Warningf("failed to get the RTE DaemonSet: %v", err)
 					return false
 				}
+				recreated := dsInitialTimeStamp.Before(&ds.CreationTimestamp)
```

Comment:
better to check the Generation: https://github.com/openshift-kni/numaresources-operator/blob/e2e-infra-numacell/test/utils/objects/wait/daemonset.go#L60 because, well, this is the recommended k8s way to ensure an object was updated by somewhere in the cluster, and because this way should be (more) resistant against clock skew (if any) between nodes

Reply:
ok, makes sense.